src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1019,15 +1020,17 @@ struct compute_dbox_type<tmpl::list<ItemsPack...>, ComputeItemsList> {
  * \brief Get all the Tags that are compute items from the `TagList`
  */
 template <class TagList>
-using get_compute_items = tmpl::filter<TagList, db::is_compute_tag<tmpl::_1>>;
+using get_compute_items =
```

Comment:
Should this metafunction and the one below also be renamed? In particular, in the docs for the function below I don't understand what "all Tags that are items" means.

Reply:
yes, I was planning to do that and forgot